loader.qml

```diff
@@ -1,57 +1,193 @@
+                            id: daysTumbler
+                            enabled: settime.checked
+                            currentIndex: 24
+                            model: 31
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                            delegate: Label {
+                                text: index + 1
+                                horizontalAlignment: Text.AlignHCenter
+                                verticalAlignment: Text.AlignVCenter
+                            }
+                        }
+                    }
+                }
+                Frame {
+                    padding: 0
+                    Row {
+                        Tumbler {
+                            id: hoursTumbler
+                            enabled: settime.checked
+                            currentIndex: 16
+                            model: 24
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                        }
+                        Tumbler {
+                            id: minutesTumbler
+                            enabled: settime.checked
+                            currentIndex: 58
+                            model: 60
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                        }
+                        Tumbler {
+                            id: secondsTumbler
+                            enabled: settime.checked
+                            currentIndex: 28
+                            model: 60
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                        }
+                    }
+                }
             }
-            Item { Layout.fillWidth: true }
-            Button {
-                id: screenshot
-                text: "Screenshot"
-                onClicked: myFrame.snapshot()
+            RowLayout {
+                Text {
+                    leftPadding: 100
+                    text: "featureSlider"
+                }
+                Slider {
+                    id: featureSlider
+                    width: 700
+                    from: 0
+                    value: 0.5
+                    to: 1
+                    Repeater {
+                        model: 25
+                        delegate: Rectangle {
+                            x: parent.horizontalPadding + parent.availableWidth * index / 24
+                            y: parent.height - height
```

Comment:
isn't that anchors.bottom: parent.bottom ?

Reply:
I didn't understand what you meant at first, but yes, I can use that instead of the definition for `y`.